detections/endpoint/dns_exfiltration_using_nslookup_app.yml

```diff
@@ -0,0 +1,56 @@
+name: DNS Exfiltration Using Nslookup App
+id: 2452e632-9e0d-11eb-bacd-acde48001122
+version: 1
+date: '2021-04-15'
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: this search is to detect potential DNS exfiltration using nslookup application.
+  This technique are seen in couple of malware and APT group to exfiltrated collected
+  data in a infected machine or infected network. This detection is looking for unique
+  use of nslookup where it tries to use specific record type, TXT, A, AAAA, that
+  are commonly used by attacker and also the retry parameter which is designed to
+  query C2 DNS multiple tries.
+search: '| tstats `security_content_summariesonly` values(Processes.process) as cmdline
```

Comment:
Just a a minor thing w SPL writing: We should probably write  `values(Processes.process) as process` so that we don't introduce a new field name, same with other `values()`

Reply:
thanks bhavin will fix this one :)